packages/core/useFocus/index.md

```diff
@@ -4,15 +4,15 @@ category: Sensors
 
 # useFocus
 
-Reactive utility to track or set the focus state of a DOM element. State changes happen on `focus` and `blur` events. Setting reactive value from the outside will trigger `.focus()` and `blur()` events for `true` and `false` values respectively.
+Reactive utility to track or set the focus state of a DOM element. State changes to reflect whether the target element is the focused element. Setting reactive value from the outside will trigger `focus` and `blur` events for `true` and `false` values respectively.
 
 ## Basic Usage
 
 ```ts
 import { useFocus } from '@vueuse/core'
 
-const focusableElement = ref();
-const { focused } = useFocus({ target: focusableElement })
+const target = ref();
```

Comment:
If we change this name then let's add an additional info somewhere in the docs that only *focusable* elements will work with this composable. Let's spare potential users headache of "why it's not working" 😄

Reply:
> If we change this name then let's add an additional info somewhere in the docs that only _focusable_ elements will work with this composable. Let's spare potential users headache of "why is it not working" 😄

I'm following this PR keenly as I have run into an issue where I keep getting this error:
```
ERROR in ./node_modules/@vueuse/integrations/useFocusTrap.mjs 3:0-45
Module not found: Error: Can't resolve 'focus-trap' in '/home/teena/projects/****/node_modules/@vueuse/integrations'
```

So If I understood you correctly, you can only use this on focusable elements?

I'm building a modal, and will like to trap the focus in the modal when it is opened.

Reply:
@teena-dot-dev I believe the error you are seeing is actually the result of a missing dependency. The `useFocusTrap` composable is just a reactive wrapper around the [focus-trap](https://github.com/focus-trap/focus-trap) library so you likely just need to install that dependency in order for it to work correctly.

For `useFocusTrap`, the target element does not actually need to be focusable. See it's documentation for more information on how it decides which element to focus when the focus trap is enabled.

Reply:
> @teena-dot-dev I believe the error you are seeing is actually the result of a missing dependency. The `useFocusTrap` composable is just a reactive wrapper around the [focus-trap](https://github.com/focus-trap/focus-trap) library so you likely just need to install that dependency in order for it to work correctly.
> 
> For `useFocusTrap`, the target element does not actually need to be focusable. See its documentation for more information on how it decides which element to focus when the focus trap is enabled.

So I'd need to install `focus-trap`?
It's strange because I used it before without errors. I recently updated my dependencies and all of a sudden I'm faced with this.
Thanks though.
I'll try it now and if it does work, I'll open an issue with more details

Reply:
Works again, thanks!

Reply:
It's a bit of a pity that nobody responded me on this one :(